scipy/special/_add_newdocs.py

```diff
@@ -883,6 +898,11 @@ def add_newdoc(name, doc):
     -----
     .. versionadded:: 1.11.0
 
+    Like ``betainc``, ``betaincc(a, b, x)`` is treated as a two parameter
+    family of functions of a single variable `x`, rather than as a function of
+    three variables. See the ``betainc`` docstring for more info on how this
```

Comment:
Maybe if we want to link?
```suggestion
    Like `betainc`, ``betaincc(a, b, x)`` is treated as a two parameter
    family of functions of a single variable `x`, rather than as a function of
    three variables. See the `betainc` docstring for more info on how this
```